README.md

```diff
@@ -1,58 +1,95 @@
-# Sonarr
+<p align="center">
+<img src="./Logo/256.png" alt="Sonarr">
+</p>
 
 Sonarr is a PVR for Usenet and BitTorrent users. It can monitor multiple RSS feeds for new episodes of your favorite shows and will grab, sort and rename them. It can also be configured to automatically upgrade the quality of files already downloaded when a better quality format becomes available.
 
-## Major Features Include:
+## Getting Started
 
-* Support for major platforms: Windows, Linux, macOS, Raspberry Pi, etc.
-* Automatically detects new episodes
-* Can scan your existing library and download any missing episodes
-* Can watch for better quality of the episodes you already have and do an automatic upgrade. *eg. from DVD to Blu-Ray*
-* Automatic failed download handling will try another release if one fails
-* Manual search so you can pick any release or to see why a release was not downloaded automatically
-* Fully configurable episode renaming
-* Full integration with SABnzbd and NZBGet
-* Full integration with Kodi, Plex (notification, library update, metadata)
-* Full support for specials and multi-episode releases
-* And a beautiful UI
+[![Installation](https://img.shields.io/badge/wiki-installation-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation)
+[![Docker](https://img.shields.io/badge/wiki-docker-1488C6.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+[![FAQ](https://img.shields.io/badge/wiki-FAQ-BF55EC.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/wiki/FAQ)
+
+- [Install Sonarr for your desired OS](https://github.com/Sonarr/Sonarr/wiki/Installation) or use [Docker](https://github.com/Sonarr/Sonarr/wiki/Installation#Docker)
+- Connect to the UI through http://localhost:8989 or http://your-ip:8989 in your web browser
+- See the [Wiki](https://github.com/Sonarr/Sonarr/wiki) for further configuration
+
+## Downloads
+
+Type | develop (stable) |
+|---|---|
+Binary Releases | [![GitHub Releases](https://img.shields.io/badge/downloads-releases-brightgreen.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/releases)
+Docker (linuxserver.io, x86_64, arm64, armhf) | [![Docker release](https://img.shields.io/badge/linuxserver-sonarr:latest-blue.svg?colorB=1488C6&maxAge=60&style=flat-square)](https://store.docker.com/community/images/linuxserver/sonarr)
+
+
+## Support
+
+[![Donate](https://img.shields.io/badge/donate-bitcoin-gold.svg?maxAge=60&style=flat-square)](https://sonarr.tv/donate)
+
+[![Discord](https://img.shields.io/badge/discord-chat-7289DA.svg?maxAge=60&style=flat-square)](https://discord.gg/M6BvZn5)
+[![Reddit](https://img.shields.io/badge/reddit-discussion-FF4500.svg?maxAge=60&style=flat-square)](https://www.reddit.com/r/sonarr)
+[![GitHub](https://img.shields.io/badge/github-issues-red.svg?maxAge=60&style=flat-square)](https://github.com/Sonarr/Sonarr/issues)
```

Comment:
Issues are not for support, it should be dropped from here or the section renamed (also again issues is at the top of the page, I don’t see a reason to have it in the reader too).